bodhi/server/util.py

```diff
@@ -437,7 +437,7 @@ def type2icon(context, kind):
         'enhancement': 'fa-bolt',
     }.get(kind)
 
-    return "<span class='label label-%s'><i class='fa %s'></i></span>" % (cls, fontawesome)
+    return "<span class='label label-%s' data-toggle='tooltip' title='%s update'><i class='fa %s'></i></span>" % (cls, kind, fontawesome)
```

Comment:
This pull request also fails an automated style checking test that is asserting that no code is more than 100 columns wide. It is upset about this line:

```
/home/vagrant/bodhi/bodhi/server/util.py:440:101: E501 line too long (137 > 100 characters)
F.
======================================================================
FAIL: Enforce PEP-8 compliance on the codebase.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vagrant/bodhi/bodhi/tests/test_style.py", line 37, in test_code_with_flake8
    self.assertEqual(subprocess.call(flake8_command), 0)
AssertionError: 1 != 0
```